src/Compress.jl

```diff
@@ -73,14 +73,11 @@ end
 
 function compress(path::String, uncompressed::Dict,
                   versions::Vector{VersionNumber} = load_versions(path))
-    inverted = Dict()
+    inverted = Dict{Pair{String,String},Vector{VersionNumber}}()
     for (ver, data) in uncompressed, (key, val) in data
-        if !STDLIB_TOML
-            val isa TOML.TYPE || (val = string(val))
-        end
-        push!(get!(inverted, key => val, VersionNumber[]), ver)
```

Comment:
Turns out that we need to stringify already here since the dict keys will be different when some input UUIDs are `String` and some `Base.UUID`...